packages/main/tests/robot/test_robocorp_workitems.robot

```diff
@@ -3,7 +3,9 @@ Suite setup    Load mock library
 Library        OperatingSystem
 
 *** Variables ***
-${RESOURCES}     ${CURDIR}/../resources
+${RESOURCES}    ${CURDIR}/../resources
+${temp_in}      ${RESOURCES}/temp_items.json
```

Comment:
Are these files gitignored?

Reply:
Yes, there's a _resources/.gitignore_ with `temp_*.json` and makes more sense to move the output workitems in _results/_.